service/src/main/scala/org/apache/celeborn/server/common/http/HttpUtils.scala

```diff
@@ -50,7 +51,7 @@ object HttpUtils {
         url.getQuery
           .split("&")
           .map(_.split("="))
-          .map(arr => arr(0).toUpperCase(Locale.ROOT) -> arr(1).toUpperCase(Locale.ROOT)).toMap
+          .map(arr => arr(0).toUpperCase(Locale.ROOT) -> arr(1).toLowerCase(Locale.ROOT)).toMap
```

Comment:
why parameter value need upper/lower case? It seems easy to make mistakes in general logic